data/projects.json

```
@@ -289,5 +289,15 @@
         "site": "https://ecommerce-multi-page.vercel.app/index.html",
         "tags": "e-commerce, html, css, vercel"
     }
+   {
+        "name": "clash-of-space",
+        "display": "Clash of Space",
+        "description": "This is a space-craft war and alien shooting game",
+        "author": "Rajspeaks",
+        "category": "game",
+        "level": "basic",
+        "site": "https://rajspeaks.github.io/clash-of-space/",
+        "tags": "game, html, css, javascript, space"
```

To a match the project to a idea please add the first tag of the idea, in this case
```suggestion
        "tags": "game, html, css, javascript, space, clash-of-space"
```